start-site/src/main/resources/application.yml

```diff
@@ -1050,16 +1060,46 @@ initializr:
           description: Provides client-side support for externalized configuration in a distributed system. Using HashiCorp's Vault you have a central place to manage external secret properties for applications across all environments.
           groupId: org.springframework.cloud
           artifactId: spring-cloud-starter-vault-config
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-vault/docs/current/reference/html/#_quick_start
```

Comment:
There's an extra `_` in the anchor that makes this link invalid.

Reply:
So that quickstart is exactly what I had in mind. If you think the two other links are important, I'd rather update that section to add them and add only this link here.